scripts/resolve_network.py

```diff
@@ -23,28 +23,32 @@
 
 def add_H2(n, snakemake):
 
-    area = snakemake.config['area']
     year = snakemake.wildcards.year
-    policy = snakemake.wildcards.policy
     country_targets = snakemake.config[f"h2_target_{year}"]
     
     for country, target in country_targets.items():
-        
-        node = geoscope(n, country, area)['node']
-    
-        # remove electricity demand of electrolysis
-        load_i = pd.Index([f"{node} electrolysis demand"])
-        if load_i[0] in n.loads.index:
-            n.mremove("Load", load_i)
-        if policy == "ref":
+        nodes = n.buses[(n.buses.index.str[:2]==country) & (n.buses.country != '')].index
+        if nodes.empty: 
             continue
-        
-        add_H2_node(n, snakemake, node, target)
+        elif country == 'DK':
```

Comment:
would be better to have not a hard coded solution, e.g. "GB" has also two nodes. My suggestion something like
```
country_counts = n.buses[n.buses.carrier == "AC"].country.value_counts()
fraction = 1/n.buses['country'].map(country_counts)
add_H2_node(n, snakemake, nodes[0], fraction.loc[nodes]*target)
```